dask/multiprocessing.py

```diff
@@ -171,13 +173,22 @@ def get(
         If True [default], `fuse` is applied to the graph before computation.
     pool : Executor or Pool
         Some sort of `Executor` or `Pool` to use
+    initializer: function
+        Ignored if ``pool`` is not ``None``.
+        Function to initialize a worker process before running any tasks in it.
+    replace_default_initializer: bool
+        Ignored if ``initializer`` is ``None``.
+        If False [default], function `initializer` will be called immediately after
+        the default process-initializer is run.
+        If True, function `initializer` will replace the default process-initializer.
     chunksize: int, optional
         Size of chunks to use when dispatching work.
         Defaults to 5 as some batching is helpful.
         If -1, will be computed to evenly divide ready work across workers.
     """
     chunksize = chunksize or config.get("chunksize", 6)
     pool = pool or config.get("pool", None)
+    initializer = initializer or config.get("multiprocessing.initializer", None)
```

Comment:
Do we want to check for `replace_default_initializer` in `dask.config` as well?

Reply:
That is done later in line 209 (if that's what you mean):
```python
        initializer = partial(
            initialize_worker_process,
            user_initializer=initializer,
            override=(
                replace_default_initializer
                or config.get("multiprocessing.replace_initializer", False)
            )
        )
```

Reply:
Also related question what should we do if `initializer` is not set, but `replace_initializer` is?

Reply:
I figured in that case we'd just ignore `replace_initializer`.
In fact, I added a line to the docstring for `multiprocessing.get()` which says:
```python
    replace_default_initializer: bool
        Ignored if ``initializer`` is ``None``.
```